src/decorator/string/IsStrongPassword.ts

```diff
@@ -0,0 +1,64 @@
+import { ValidationOptions } from '../ValidationOptions';
+import { buildMessage, ValidateBy } from '../common/ValidateBy';
+import isStrongPasswordValidator from 'validator/lib/isStrongPassword';
+
+export const IS_STRONG_PASSWORD = 'isStrongPassword';
+
+/**
+ * Options to be passed to IsStrongPassword decorator.
+ */
+export interface IsStrongPasswordOptions {
+  /**
+   * The minimum password length.
+   */
+  minLength?: number;
+
+  /**
+   * The minimum amount of lowercase letters.
+   */
+  minLowercase?: number;
+
+  /**
+   * The minimum amount of uppercase letters.
+   */
+  minUppercase?: number;
+
+  /**
+   * The minimum amount of numbers.
+   */
+  minNumbers?: number;
+
+  /**
+   * The minimum amount of symbols.
```

Comment:
```suggestion
   * The minimum amount of special characters.
```

I think this is the more common terminology for special symbols like`#`, `%`, and so on.

However, I can understand the desire to keep the consistent naming convention since this is referred to as "symbols" in validator.js